util.py

```diff
@@ -149,6 +149,19 @@ def make_dataloader(dataset_id, input_ids, labels, attention_masks, batch_size=1
                             )
     return dataloader
 
+def split_episode(batch, ratio=.5):
```

Comment:
Could this be no longer needed? I don't see where it's being used.

Reply:
yes can be deleted! I think it was already in an earlier version